compiler/semfold.nim

```diff
@@ -211,12 +211,7 @@ proc evalOp(m: TMagic, n, a, b, c: PNode; g: ModuleGraph): PNode =
   of mUnaryMinusF64: result = newFloatNodeT(- getFloat(a), n, g)
   of mNot: result = newIntNodeT(1 - getInt(a), n, g)
   of mCard: result = newIntNodeT(nimsets.cardSet(g.config, a), n, g)
-  of mBitnotI:
```

Comment:
Huh, reverting these bugfixes?

Reply:
If the narrowing casts are now correctly folded we don't need to perform them by hand after every operations